tesla-hero-slider/src/components/slide-params.js

```diff
@@ -0,0 +1,86 @@
+import React, {Component} from 'react';
+import propTypes from 'prop-types';
+import AnimateValue from './animate-value';
+import './../styles/scss/components/slide-params.css';
+
+
+class SlideParams extends Component {
+
+	static propTypes = {
+		activeCar: propTypes.object.isRequired,
+		animationForward: propTypes.bool.isRequired
+	};
+
+	state = {
+		delayTopSpeed: 0,
+		delayMph: 0,
+		delayMileRange: 0
+	};
+
+	componentWillReceiveProps(props) {
+		let delay = 500,
+			delayTopSpeed = delay,
+			delayMph = delay + 250,
+			delayMileRange = delay + 500;
+
+		if (!props.animationForward) {
+			delayTopSpeed = delay + 500;
+			delayMileRange = delay;
+		}
+
+		this.setState({
+			delayTopSpeed,
+			delayMph,
+			delayMileRange
+		})
+	}
+
+	render() {
+		let activeCar = this.props.activeCar,
```

Comment:
Use const if the value is not going to change. It will prevent common bugs. Readability is a debatable issue, but I'd suggest using destruction. So you can convert 39/40 to:

```js
const { activeCar, animationForware } = this.props;
```